helpers/misc/editBotVoiceState.ts

```diff
@@ -0,0 +1,26 @@
+import { Bot } from "../../bot.ts";
+
+/** Updates the current user's voice state. `channelId` must currently point to a stage channel. Bot must already have joined `channelId`. You must have the `MUTE_MEMBERS` permission to unsuppress yourself. You can always suppress yourself. You must have the `REQUEST_TO_SPEAK` permission to request to speak. You can always clear your own request to speak. You are able to set `requestToSpeakTimestamp` to any present or future time.
```

Comment:
Needs jsdocs like other helpers

Reply:
@Skillz4Killz Sorry what? Isn't that JSDocs comments?